updatecli/updatecli.d/debian.yaml

```diff
@@ -0,0 +1,76 @@
+---
+name: Bump Debian Bookworm version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  bookwormLatestVersion:
+    kind: dockerimage
+    name: "Get the latest Debian Bookworm Linux version"
+    spec:
+      image: "debian"
+      tagFilter: "bookworm-*"
+      versionfilter:
+        kind: regex
+        pattern: >-
+          bookworm-\d+$
+
+conditions:
+  testDockerfileArg:
+    name: "Does the Dockerfile have an ARG instruction for the Debian Bookworm Linux version?"
+    kind: dockerfile
+    disablesourceinput: true
+    spec:
+      file: debian/Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "DEBIAN_RELEASE"
+  testVersionInBakeFile:
+    name: "Does the bake file have variable DEBIAN_RELEASE?"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: docker-bake.hcl
+      matchpattern: "(.*DEBIAN_RELEASE.*)"
+
```

Comment:
Are these conditions mandatory?

As the targets will fail if there is no file to update (I think?), I would skip them.

```suggestion
```

Reply:
Thanks for your review, Hervé. 🙏 
I think it would work, but would maybe prove more difficult for me to maintain later on. 😊